pkg/azure/ocpgwdeployer.go

```diff
@@ -267,7 +267,7 @@ func (d *ocpGatewayDeployer) initMachineSet(name, zone string) (*unstructured.Un
 }
 
 func (d *ocpGatewayDeployer) deployGateway(zone string) error {
-	machineSet, err := d.initMachineSet(MachineName(d.azure.InfraID, d.azure.Region, zone), zone)
+	machineSet, err := d.initMachineSet(MachineName(d.azure.Region, zone), zone)
```

Comment:
If we just need any valid, unique name for the machine set, why not just use a UUID or use the K8s generate name functionality?